src/openforms/contrib/microsoft/client.py

```diff
@@ -83,7 +88,9 @@ def upload_bytes(self, bytes_: bytes, remote_path: Union[str, Path]):
     def upload_stream(self, stream, stream_size: int, remote_path: Union[str, Path]):
         return self.target_folder.upload_file(
             None,
-            remote_path,
+            # upload_file says it accepts strings or Path objects
+            # (https://github.com/O365/python-o365/blob/master/O365/drive.py#L1239), but Path objects give errors
+            remote_path if isinstance(remote_path, str) else str(remote_path),
```

Comment:
```suggestion
            str(remote_path),
```
if it's already a string, this is a noop